vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/VLogReasoner.java

```diff
@@ -809,4 +809,4 @@ void setReasonerState(ReasonerState reasonerState) {
 		this.reasonerState = reasonerState;
 	}
 
-}
+}
```

Comment:
There is no new line at the end of the file VLogReasoner. Please add the newline back.